docs/markdown/Reference-tables.md

```diff
@@ -46,6 +46,7 @@ These are return values of the `get_linker_id` method in a compiler object.
 | ld.bfd     | The GNU linker                              |
 | ld.gold    | The GNU gold linker                         |
 | ld.lld     | The LLVM linker, with the GNU interface     |
+| ld.mold    | A Modern Linker                             |
```

Comment:
I know `"A Modern Linker"` is the slogan for mold, but most of these are more descriptive. Maybe a "High performance GNU like linker"?

Reply:
Replaced by `The fast MOLD linker`.

I thought 'GNU like' is not saying anything specific (well it has the same interface, but unlike LLVM's linker it has just this one interface, so why mention it?

High performance was a bit too adverting speak for me, so downgraded to fast ;-)

I used the 'slogan' because that is the name/abbreviation expanded.